frontend/src/App.js

```diff
@@ -35,6 +37,19 @@ function App() {
         })}/>
     };
 
+    const renderWarehouse = () => {
+        return <ProtectedComponent conditions={user} render={(() => {
```

Comment:
Is 'user' condition enough? Do we need to add user.role === "ROLE_SYSTEM_ADMIN"  for example?